procedures/foreman/branch.md.erb

```diff
@@ -1,16 +1,45 @@
+# Two weeks before branching
 
-* [ ] Announce upcoming branching to discourse development category a month before
-* [ ] Change `$latest` and `$next` parameters on [web class](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/web/manifests/init.pp) to point to the new version numbers
+* [ ] Make releases of installer modules
+  * Tier 0 (no dependencies)
+    * [ ] [certs](https://github.com/theforeman/puppet-certs)
+    * [ ] [dhcp](https://github.com/theforeman/puppet-dhcp)
+    * [ ] [dns](https://github.com/theforeman/puppet-dns)
+    * [ ] [foreman](https://github.com/theforeman/puppet-foreman)
+    * [ ] [git](https://github.com/theforeman/puppet-git)
+    * [ ] [puppet](https://github.com/theforeman/puppet-puppet)
+    * [ ] [qpid](https://github.com/theforeman/puppet-qpid)
+    * [ ] [tftp](https://github.com/theforeman/puppet-tftp)
+  * Tier 1 (Dependencies on Tier 0)
+    * [ ] [candlepin](https://github.com/theforeman/puppet-candlepin)
+    * [ ] [foreman_proxy](https://github.com/theforeman/puppet-foreman_proxy)
+    * [ ] [pulp](https://github.com/theforeman/puppet-pulp)
+  * Tier 2 (Dependencies on Tier 1)
+    * [ ] [foreman_proxy_content](https://github.com/theforeman/puppet-foreman_proxy_content)
+    * [ ] [katello](https://github.com/theforeman/puppet-katello)
+* [ ] Ensure headline features planned for the release have been merged or push them off to the next release.
+* Translations:
+  * [ ] Update locales in foreman **develop**: `make -C locale tx-update`
+  * [ ] Ask plugin authors to start extracting i18n strings and pushing the changes into develop/master git branches so Transiflex can pick it up
+  * [ ] Announce string freeze date on discourse and send announcement via https://www.transifex.com/foreman/foreman/announcements/
+  * [ ] Update [foreman-dev](https://community.theforeman.org/c/development) with [translations status](https://www.transifex.com/projects/p/foreman/) to encourage 100% translations before release
+
+# During the week before branching
+
+* [ ] Announce start of stabilization week to discourse [development category](https://community.theforeman.org/c/development).
+* Prepare the manual for the new version:
+  * [ ] Change `$latest` and `$next` parameters on [web class](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/web/manifests/init.pp) to point to the new version numbers
+  * [ ] Copy [website manual content](https://github.com/theforeman/theforeman.org/tree/gh-pages/manuals) from nightly to <%= release %> and update version numbers mentioned in it.
+  * [ ] Clean up deprecation and upgrade warnings from nightly manual.
 * [ ] Refresh unattended templates
   * [ ] Sync the community-templates to Foreman core, by running `script/sync_templates.sh`
   * [ ] Create a <%= @release %>-stable branch in [community-templates](https://github.com/theforeman/community-templates)
+* [ ] Update puppet module versions in installer to the newly released versions
```

Comment:
We have nightly builds so they should automatically be pulled in. During branching there's already a task to pin to released versions which automatically takes the latest ones.

Reply:
removed